README.md

```diff
@@ -129,9 +129,20 @@ curl -fsSL "https://github.com/lima-vm/lima/releases/download/${VERSION}/lima-${
 
 - To install Lima from the source, run `make && make install`.
 
+- To change the build configuration, run `make config` or `make menuconfig`.
+  This requires kconfig tools installed, it is also possible to edit `.config`.
+
 > **NOTE**
 > Lima is not regularly tested on ARM Mac (due to lack of CI).
 
+##### Kconfig tools
+
+The tools are available as either "kconfig-frontends" or "kbuild-standalone".
+There is one `conf` for the text, and one `mconf` for the menu interface.
+
+A python implementation is available at <https://pypi.org/project/kconfiglib>.
+It can be installed with `pip install --user kconfiglib`, including `guiconfig`.
+
```

Comment:
This should be now in https://lima-vm.io/docs/installation/

Reply:
I think you mean `website/content/en/docs/Installation/`

Reply:
Done